client/v3/kubernetes/client.go

```diff
@@ -31,24 +31,20 @@ func New(cfg clientv3.Config) (*Client, error) {
 	}
 	kc := &Client{
 		Client: c,
-		kv:     clientv3.RetryKVClient(c),
 	}
-	kc.Kubernetes = kc
 	return kc, nil
 }
 
 type Client struct {
 	*clientv3.Client
-	Kubernetes Interface
```

Comment:
Why you remove `Kubernetes` now? If you remove it, then you need to provide another way to return a `Kubernetes` interface to client.